pkg/util/util.go

```diff
@@ -124,10 +131,16 @@ func Until(f func(), period time.Duration, stopCh <-chan struct{}) {
 			defer HandleCrash()
 			f()
 		}()
+
+		jitteredPeriod := period
+		if jitterFactor > 0.0 {
```

Comment:
This is the first PR. The second is intended to replace use of util.Until with util.JitterUtil in some sync loops in kubelet with positive jitterFactor